.github/workflows/lint_python.yml

```diff
@@ -0,0 +1,25 @@
+name: lint_python
+on: [pull_request, push]
+jobs:
+  lint_python:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-python@v2
+      - run: pip install --upgrade pip wheel
+      - run: pip install bandit codespell darker flake8 flake8-2020 flake8-bugbear
+                         flake8-comprehensions isort mypy pytest pyupgrade safety
+      - run: bandit --recursive --skip B101 . || true  # B101 is assert statements
+      - run: darker --check .
+      - run: codespell || true  # --ignore-words-list="" --skip="*.css,*.js,*.lock"
+      - run: flake8 . --count --select=E9,F63,F7,F82 --show-source --statistics
+      - run: flake8 . --count --exit-zero --max-complexity=10 --max-line-length=88
+                      --show-source --statistics
+      - run: isort --check-only --profile black . || true
```

Comment:
`isort` is already being run using [pytest-darker](https://pypi.org/project/pytest-darker/) as well, see
- [`pytest.ini`](https://github.com/akaihola/darker/blob/8a1835c952b9743ea5cb6d40d23eb9c6458e7828/pytest.ini#L7)
- [`pyproject.toml`](https://github.com/akaihola/darker/blob/8a1835c952b9743ea5cb6d40d23eb9c6458e7828/pyproject.toml#L8-L10) (`[tool.isort]` configuration section)

If we'd run `isort` explicitly instead of through [pytest-darker](https://pypi.org/project/pytest-darker/), I'd remove `--isort` from [`pytest.ini`](https://github.com/akaihola/darker/blob/8a1835c952b9743ea5cb6d40d23eb9c6458e7828/pytest.ini#L7), and rely on [`pyproject.toml`](https://github.com/akaihola/darker/blob/8a1835c952b9743ea5cb6d40d23eb9c6458e7828/pyproject.toml#L8-L10) for `isort` configuration instead of command line options here. This way running `isort` or `darker --isort` manually would use the same options.